### PR TITLE
:adhesive_bandage: Combobox: placeholder fix

### PR DESCRIPTION
**Pull request by larseirikhansen**
### Description

Original issue afaik: https://nav-it.slack.com/archives/C7NE7A8UF/p1714377788972099

With this fix, the placeholder of the Combobox input-element is no longer visible if the user has added a value to selectedOptions (i.e. picked a value from the dropdown or added a custom value). This took a while because of internal discussions of the placeholder prop in Combobox.

### CODE REVIEW

File: @navikt/core/react/src/form/combobox/combobox.stories.tsx [hunk 1]
```diff
@@ -35,7 +35,11 @@ const options = [
 ];
 
 export const Default: StoryFunction = (props) => (
-  <UNSAFE_Combobox {...props} id="combobox" />
+  <UNSAFE_Combobox
+    {...props}
+    maxSelected={{ limit: Number(props.maxSelected) }}
```
Comment (it-vegard) [inline @navikt/core/react/src/form/combobox/combobox.stories.tsx:40]: Hvorfor maxSelected på default-eksempelet?

File: @navikt/core/react/src/form/combobox/Input/InputController.tsx
Comment (HalvorHaugan) [inline @navikt/core/react/src/form/combobox/Input/InputController.tsx:1]: There is already a PR by Julian covering this: https://github.com/navikt/aksel/pull/3071/files
However, the main focus in that PR is not about the placeholder, so I think it's better to have a separate PR on it anyways. But I wonder if Julian's solution is more elegant? He did it in Input.tsx.

File: .changeset/friendly-pens-sin.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@navikt/ds-react": patch
+"@navikt/aksel": patch
```
Comment (HalvorHaugan) [inline .changeset/friendly-pens-sin.md]: Is `@navikt/aksel` affected?

File: .changeset/friendly-pens-sin.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+---
+
+:adhesive_bandage: Placeholder goes away when value added
```
Comment (HalvorHaugan) [inline .changeset/friendly-pens-sin.md]: Should mention "Combobox", e.g.
```suggestion
:adhesive_bandage: Combobox: Placeholder goes away when an option is selected
```

File: @navikt/core/react/src/form/combobox/combobox.stories.tsx (hunk 1 above)
Comment (larseirikhansen) [reply @navikt/core/react/src/form/combobox/combobox.stories.tsx:40]: Tried to test edge cases, like having that prop active in Storybook. But it didn't work from the Storybook-controls, so that was the fix 👍

File: .changeset/friendly-pens-sin.md (hunk 2 above)
Comment (larseirikhansen) [reply .changeset/friendly-pens-sin.md]: No, no it is not.

File: @navikt/core/react/src/form/combobox/Input/InputController.tsx
Comment (larseirikhansen) [reply @navikt/core/react/src/form/combobox/Input/InputController.tsx:1]: Agreed, will change it.

**Review by HalvorHaugan — APPROVED** [review]
